app/assets/javascripts/module/tree.js.coffee

```diff
@@ -1,5 +1,8 @@
 class Module.Tree
   @load: (loading_html, module_id) ->
+    # FIXME: The messages we send on loading_html are the same already
```

Comment:
I'm not sure. Even if the user ask for a reprocess, the page still loads the correct div anyway.